app/gui2/ydoc-server/index.ts

```diff
@@ -21,7 +21,7 @@ type ConnectionData = {
   user: string
 }
 
-export async function createGatewayServer(httpServer: Server, rustFFIPath: string) {
+export async function createGatewayServer(httpServer: Server, rustFFIPath?: string) {
```

Comment:
Why this change? Anyone running server will do that probably in node, and in that case it won't work without path to rustFFI wasm.

Reply:
This is just so the type error goes away. The usage in `gatewayServer` function in vite config is not using that parameters. Since it is truly optional, I just made the types match reality.